packages/stem-begroot/src/step-4/index.tsx

```diff
@@ -18,16 +19,20 @@ export const Step4 = ({
     <>
       {header}
       <Spacer size={1.5} />
-      <h5>{voteMessage}</h5>
-      <p>{thankMessage}</p>
+      <Heading5>{voteMessage}</Heading5>
+      <Paragraph>{thankMessage}</Paragraph>
 
       {showNewsletterButton ? (
-        <SecondaryButton
-          onClick={() => {
-            // What should happen here?
-          }}>
-          Hou mij op de hoogte
-        </SecondaryButton>
+        <>
+          <Spacer size={2} />
```

Comment:
Is deze `Spacer` hier nodig?

Reply:
Ja, tenzij we het met css oplossen. Maar het zorgt ervoor dat de button niet te dicht op de tekst staat.